### PR TITLE
Expose the zap.NewNop function

### PR DESCRIPTION
**Pull request by mattrobenolt**

### CODE REVIEW

File: logger.go [hunk 1]
```diff
@@ -18,7 +18,9 @@ func NewPlanetScaleLogger() *zap.Logger {
 }
 
 // New creates a new default PlanetScale Logger.
-var New = NewPlanetScaleLogger
+func New() *zap.Logger {
```
Comment (mattrobenolt) [inline logger.go:21]: Turns out, these are slightly better as actual function calls even though they are identical. It helps godoc out so it's an actual function, and not a variable.

File: logger.go [hunk 2]
```diff
@@ -27,6 +29,11 @@ func NewPlanetScaleSugarLogger() *zap.SugaredLogger {
 	return NewPlanetScaleLogger().Sugar()
 }
 
+// NewNop returns a no-op logger
+func NewNop() *zap.Logger {
```
Comment (mattrobenolt) [inline logger.go:33]: It's nice to keep zap as an implicit dependency.

**Review by JTCunning — APPROVED** [review]
🍪

**Comment by joshuarli** [comment]
Great work matt.

**Pull request merged**